src/types.ts

```diff
@@ -533,6 +533,12 @@ export type ServerInteractiveMessage = {
               };
               button_reply: never;
           };
```

Comment:
There's an extra ; which should be in 541

Reply:
Great catch, thanks ✅